docker/Dockerfile

```diff
@@ -109,16 +109,10 @@ RUN git clone -b ${TORCH_BENCH_BRANCH} https://github.com/pytorch/benchmark.git
     pip install --no-cache Jinja2==3.1.2 markupsafe==2.0.1 beartype==0.15.0 mpmath==1.3.0 && \
     python install.py --continue_on_fail && cd .. && \
     echo ${TRANSFORMERS_VERSION} && if [ "${TRANSFORMERS_VERSION}" = "default" ]; then \
-        git clone https://github.com/huggingface/transformers.git; \
-        cd transformers; \
-        git submodule sync; \
-        git submodule update --init --recursive; \
-        git checkout `cat /workspace/pytorch/.ci/docker/ci_commit_pins/huggingface.txt`; \
-        python setup.py bdist_wheel;  \
-        pip install dist/*.whl; \
-        cd ../; \
+        export TRANSFORMERS_COMMIT=`cat /workspace/pytorch/.ci/docker/ci_commit_pins/huggingface.txt`; \
+        pip install --force-reinstall git+https://github.com/huggingface/transformers@${TRANSFORMERS_COMMIT}; \
```

Comment:
let's also align this method in bisect search, and then test this PR with a bisect search test

Reply:
Test case is running: https://inteltf-jenk.sh.intel.com/user/weizhuo/my-views/view/weizhuoz_view/job/inductor_aws_guilty_commit_search_zwz/33/